src/Core/Streams/CallableStream.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace AsyncAws\Core\Streams;
```

Comment:
Can't we use `Stream` as namespace? 

Using singular is a good practice.

Reply:
Hm.. I see that we use `Signers` too.

Reply:
Fixed (Signers will follow in a dedicated PR)